README.md

```
@@ -37,12 +37,59 @@ This repository contains a sleek, reactive frontend, but it leans on a robust ba
+      - name: Marmelade & Eingemachtes
+        icon: mdi:fruit-cherries
+        filter_key: marmelade
+      - name: Kühltruhe (Keller)
+        icon: mdi:fridge
+        filter_key: kuhltruhe_k
+      - name: Kühltruhe (Garage)
+        icon: mdi:garage
+        filter_key: kuhltruhe_g
+```
+
 ##  Backend Setup (Required)
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)